OrbitGl/TimeGraphLayout.cpp

```diff
@@ -80,14 +81,24 @@ bool TimeGraphLayout::DrawProperties() {
   FLOAT_SLIDER_MIN_MAX(m_TextZ, -1.f, 1.f);
   FLOAT_SLIDER_MIN_MAX(m_TrackZ, -1.f, 1.f);
   FLOAT_SLIDER(m_ToolbarIconHeight);
+  FLOAT_SLIDER(scale_);
```

Comment:
FLOAT_SLIDER_MIN_MAX(scale_, 0.f, 10.f);

Reply:
This was just to show that we could specify a custom range.  Not at all critical... :-)